<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="veh" default="david" doc="Name of vehicle. ex: megaman"/>
    <arg name="local" default="true" doc="true for running everything you possibly can on laptop. false for everything running on vehicle."/>
    <!-- <arg name="config" default="baseline" doc="Specify a config."/> -->
    <!-- <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." /> -->
    <!-- <arg name="testing" default="false" doc="using for test purposes." /> -->
    <arg name="pkg_name" value="lane_filter"/>
    <arg name="node_name" default="lane_pose_visualizer_node"/>

    <group ns="$(arg veh)">        
        <!-- run local -->
        <remap from="~lane_pose" to="lane_filter_node/lane_pose"/>
        <node if="$(arg local)" name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" clear_params="true" required="true"/>

        <!-- run remote -->
        <node unless="$(arg local)" machine="$(arg veh)" name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" clear_params="true" required="true"/>
    </group>

    <!-- Publications -->

    <!-- Subscriptions -->

    
</launch>
